src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/model/commonplatform/CommonPlatformHearing.java

```diff
@@ -7,17 +7,28 @@
 import lombok.NoArgsConstructor;
 import uk.gov.justice.probation.courtcasematcher.model.domain.CourtCase;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
 import java.util.List;
 
 @Data
 @Builder
 @NoArgsConstructor(access=AccessLevel.PRIVATE, force = true)
 @AllArgsConstructor
 public class CommonPlatformHearing {
+    @NotBlank
     private final String id;
+    @NotNull
+    @Valid
     private final CourtCentre courtCentre;
+    @NotNull
```

Comment:
Would it be OK to have this as NotEmpty ? I'm wondering what we'd do if there were no elements in the hearingDays collection.